src/api/v2/quickPickWizard/RecursiveQuickPickStep.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IWizardOptions } from '@microsoft/vscode-azext-utils';
+import { ResourceModelBase } from '../v2AzureResourcesApi';
+import { GenericQuickPickStep } from './GenericQuickPickStep';
+import { QuickPickWizardContext } from './QuickPickWizardContext';
+
+export class RecursiveQuickPickStep<TModel extends ResourceModelBase> extends GenericQuickPickStep<TModel> {

```

Comment:
Since this inherits `getPicks` and `prompt` from `GenericQuickPickStep`, would the `getSubWizard` interfere with this [block of code?](https://github.com/microsoft/vscode-azureresourcegroups/pull/328/files#diff-fafee4fc40399ada1bd6f49e86a9eff708bdf1230ada0e0b5614c76a03f72ac5R48-R54)